### ISSUE TITLE
Всем привет , в данный момент программа не работает !!

### ISSUE THREAD

**Issue by Levin-crk** [kiray-gd/nmocrasher#9]
Подскажите что делать , чтобы она заработала ?

**Comment by Boris20A** [comment]
Мы в Вас верим!

**Comment by Boris20A** [comment]
Сегодня в 0-30 провели техработы на сайте НМО и после этого перестала отмечать ответы

**Comment by MaximMedik** [comment]
> Подскажите что делать , чтобы она заработала ?

молиться на разраба

**Comment by Melni62** [comment]
![image](https://github.com/user-attachments/assets/e849bd91-3e4a-4efd-8e63-d37b8e826384)
настолько неумёхи что сами себе сайт ломают...

**Comment by MaximMedik** [comment]
> ![image](https://private-user-images.githubusercontent.com/179222843/361133117-e849bd91-3e4a-4efd-8e63-d37b8e826384.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjQ0OTQ5NzksIm5iZiI6MTcyNDQ5NDY3OSwicGF0aCI6Ii8xNzkyMjI4NDMvMzYxMTMzMTE3LWU4NDliZDkxLTNlNGEtNGVmZC04ZTYzLWQzN2I4ZTgyNjM4NC5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwODI0JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDgyNFQxMDE3NTlaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT0yZjVjNGRlMDhlODYyNTUyNTk2ODZlZjNjMGVjYjQ3NzE2ZDMxNDFkOTMwYzMzMzNhNTRkYjgwYTQ5YzRmYmFlJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.QaKAazNn-EnyAJO46RJB4P84Ou8Zgw-0_hhteHsiYbE) настолько неумёхи что сами себе сайт ломают...

??)

**Comment by MaximMedik** [comment]
![20240824_172207](https://github.com/user-attachments/assets/fbc921cc-9ad9-4513-9cb6-1685a550702b)
Вообще тесты не открывает, ошибку выдает

**Comment by Melni62** [comment]
Ну щас опять час-два пройдёт и снова откатят

**Comment by TemPoun** [comment]
Все что-ли не фиксанём?

**Comment by MaximMedik** [comment]
Ребят, скрипт снова решает

**Comment by Levin-crk** [comment]
> Ребят, скрипт снова решает
> 
> 

серьезно ?

**Comment by MaximMedik** [comment]
> > Ребят, скрипт снова решает
> 
> серьезно ?

У меня работает

**Comment by Melni62** [comment]
Да опять откатили) У них наладчик контужен)

**Comment by TemPoun** [comment]
Там просто сидят  гении,думаю что бутылочка водочки им сейчас не помешает

**Comment by MaximMedik** [comment]
Опять троит сайт.... Чёт замышляют...

**Comment by TemPoun** [comment]
работает?

**Comment by TemPoun** [comment]
прога парни

**Comment by TemPoun** [comment]
отпишите парни

**Comment by TemPoun** [comment]
парни

**Comment by MaximMedik** [comment]
А НМО у вас работает? Через госуслуги не могу войти
upd наладили

**Comment by MaximMedik** [comment]
Всем доброго дня, перестал отмечать, на 2х устройствах и разных аккаунтах....проблема НМО или скрипта?

**Comment by Melni62** [comment]
проблема всегда в НМО

> Всем доброго дня, перестал отмечать, на 2х устройствах и разных аккаунтах....проблема НМО или скрипта?

**Comment by Melni62** [comment]
Ребят у всех сайт уже 2-3 день как говно работает?

**Comment by MaximMedik** [comment]
Да

**Comment by vadimon8** [comment]
Ребята у вас работает программа? Запускается, но не проставляет ответы

**Comment by Fktrcf903** [comment]
На мед. психологах так. У остальных работает вроде без изменений

**Comment by vadimon8** [comment]
> На мед. психологах так. У остальных вроде работает без изменений

По разделу психиатрия также не дает? Пытался автоответ по разделу психиатрия, но не проставляет

**Comment by AndruhinP** [comment]
Добрый день. 
@kiray-gd, подскажите, пожалуйста, обрабатываются ли вопросы, которые на 24forcare повторяются более одного раза и имеют разные наборы ответов?
Спасибо.

**Comment by kiray-gd** [comment]
> Добрый день. @kiray-gd, подскажите, пожалуйста, обрабатываются ли вопросы, которые на 24forcare повторяются более одного раза и имеют разные наборы ответов? Спасибо.

Нет, события подобного рода не учтены при разработке. Скорее всего скрипт будет искать ответы в первом попавшемся варианте вопроса в массиве вопросы\ответы.

**Comment by AndruhinP** [comment]
> Ребята у вас работает программа? Запускается, но не проставляет ответы

Работает. 
Есть темы, в которых вопросы на сайте НМО звучат абсолютно не так, как на сайте 24forcare. Тут уж закат солнца вручную). Например, тема _Синдром дефицита Glut1 (по утвержденным клиническим рекомендациям) - 2024._

**Comment by KTZ-hash** [comment]
Всем привет! Подскажите в чём может быть проблема? По началу скрипт просто все вопросы на нмо скипал, после чего вышла эта ошибка. Что делать?

![b5bd30cd-f88e-449d-a39f-be9fe80da28b](https://github.com/user-attachments/assets/945b7147-85ef-44f1-96f5-5ff790e319f7)

**Comment by Melni62** [comment]
Ребят, у кого нибудь аннулировали тесты? просто думаю либо в скорости дело(делал за 0 минут) либо в регионе потому что все заявку за неделю по москве и мско аннулировали а остальниые области остались, подскажите кто сталкивался знает?

**Comment by Fktrcf903** [comment]
А ты с помощью какой программы проходил???

**Comment by belprobel** [comment]
ребят у Вас тоже сейчас не решает программа? я просто не пойму может я что то не так делаю, вроде когда на портале НМО заменили раскладку букв некоторых на английские, то решалось все, а щас эту раскладку поменяли на сайте с ответами, и у меня щас ничего не решается вообще ((((

**Comment by belprobel** [comment]
можно как то добавить в код чтобы и с русских букв переводило на английские?

**Comment by Melni62** [comment]
я сделал скрипт который исправляет этот момент, если что можете написать в телеграмм за маленькую денюжку договоримся xD @Melnik62

**Comment by MaximMedik** [comment]
> я сделал скрипт который исправляет этот момент, если что можете написать в телеграмм за маленькую денюжку договоримся xD @Melnik62

ответь в ТГ, пожалуйста

**Comment by Friendly595** [comment]
У вас тоже скипает вопросы?
UPD: Решил.

**Comment by 14karat14444** [comment]
> У вас тоже скипает вопросы? UPD: Решил.
как решил подскажи

**Comment by Nik2014a** [comment]
У кого есть возможность помочь со скриптом. Скипает ответы. Пожалуйста помогите.

**Comment by Friendly595** [comment]
> У кого есть возможность помочь со скриптом. Скипает ответы. Пожалуйста помогите.

Дайте контакты, помогу.

**Comment by Melni62** [comment]
Ребят, есть некоторые проблемы в Tamper Monkey бывает такое что скрипт резко перестаёт работать, пример:
![image](https://github.com/user-attachments/assets/b2c53920-5a8f-4a46-a0d0-954a32bd24d4) если у вас так значит проблема в этом, должно быть так: 
![image](https://github.com/user-attachments/assets/6e0126dd-0ac3-4cee-847e-15e89ca5493e)
что бы исправить эту проблему переходим по ссылке chrome://extensions/, и выключаем включаем ползунок 
![image](https://github.com/user-attachments/assets/099adc42-eaf2-495d-abdc-af230c72fa92)
Сегодня помогал уже 3-ём людям, если кто столкнётся

**Comment by Melni62** [comment]
и кстати делаю щас программу что бы можно было решать тесты которых нету на сайте 24force(потому что разработчик сайта еврей и удалил много тестов за 2020-2021 год что бы продавать из в телеграмм за 5тысяч рублей......), пока только demo-версия но может помочь кому лень)

**Comment by viktorcbr600rr** [comment]
У кого есть возможность помочь со скриптом. Скипает ответы. Пожалуйста помогите

**Comment by Friendly595** [comment]
> У кого есть возможность помочь со скриптом. Скипает ответы. Пожалуйста помогите

Могу помочь.

**Comment by viktorcbr600rr** [comment]
помогите

**Comment by Friendly595** [comment]
> помогите

Контакты можно ваши?

**Comment by viktorcbr600rr** [comment]
89232839564 тг

**Comment by Friendly595** [comment]
> 89232839564 тг

Не могу найти по номеру, можно ваш ник @...

**Comment by viktorcbr600rr** [comment]
Viktorcbr600

**Comment by wrenay09** [comment]
@Friendly595 

помогите тоже, пожалуйста, скипает вопросы
тг @Wrenay
тел 89146070699

**Comment by angelinkaaaa** [comment]
всем привет, подскажите пожалуйста, сделала все в точности как по инструкции, но когда нажимаю на кнопку включить автотест, ничего не происходит, в чем может быть проблема?
![image](https://github.com/user-attachments/assets/ef17a272-6073-407d-a612-bbadfa69e58f)
![Снимок экрана 2024-11-22 204231](https://github.com/user-attachments/assets/e036c3b8-0aa7-4d38-8b6a-ef46292e5bd1)

**Comment by kiray-gd** [comment]
@angelinkaaaa сделайте скриншот всего содержимого монитора.
А пока попробуйте стянуть темное окошко справа на середину, нажмите кнопку "Развернуть", затем включить автотест.
Можете нажать кнопку "Console" (справа сверху), там возможно будет причина по которой скрипт не работает

**Comment by angelinkaaaa** [comment]
> @angelinkaaaa сделайте скриншот всего содержимого монитора. А пока попробуйте стянуть темное окошко справа на середину, нажмите кнопку "Развернуть", затем включить автотест. Можете нажать кнопку "Console" (справа сверху), там возможно будет причина по которой скрипт не работает

![2024-11-23_20-28-33](https://github.com/user-attachments/assets/2629af5d-86bb-4030-8ec7-e5783ca0d503)

**Comment by angelinkaaaa** [comment]
> > @angelinkaaaa сделайте скриншот всего содержимого монитора. А пока попробуйте стянуть темное окошко справа на середину, нажмите кнопку "Развернуть", затем включить автотест. Можете нажать кнопку "Console" (справа сверху), там возможно будет причина по которой скрипт не работает
> 
> ![2024-11-23_20-28-33](https://private-user-images.githubusercontent.com/189388692/389215386-2629af5d-86bb-4030-8ec7-e5783ca0d503.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzIzODM0MTcsIm5iZiI6MTczMjM4MzExNywicGF0aCI6Ii8xODkzODg2OTIvMzg5MjE1Mzg2LTI2MjlhZjVkLTg2YmItNDAzMC04ZWM3LWU1NzgzY2EwZDUwMy5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQxMTIzJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MTEyM1QxNzMxNTdaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT01Yzc5YzA4OTQ3NjA0YzlmY2M0MTIzMmE5ZWZkZjBhNWM5MzJlMzg5NTIyMTNlODk4NDdmMmQwZGJmODAwMzQ4JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.GBMd0B2XFhQr2LyiB4NgZmOvju5CPpDxUlPDylEyDRI)

подскажите пожалуйста, очень плохо разбираюсь в этом

**Comment by kiray-gd** [comment]
@angelinkaaaa [image](https://github.com/user-attachments/assets/c8ff02b6-0b1b-4fba-9032-6cc3a678970c)
Перетяните темное окошко на середину.
Нажмите F5
Если вам не видно темы "итогового теста" нажмите кнопку "развернуть", затем "включить автотест".

Если это не помогло, попробуйте установить блокировщик рекламы на браузер, например adblock, затем выключить расширение tamper monkey, закрыть браузер. Затем включите браузер, включить расширение tamper monkey и попробуйте снова "решить" тест

**Comment by angelinkaaaa** [comment]
> @angelinkaaaa [image](https://github.com/user-attachments/assets/c8ff02b6-0b1b-4fba-9032-6cc3a678970c) Перетяните темное окошко на середину. Нажмите F5 Если вам не видно темы "итогового теста" нажмите кнопку "развернуть", затем "включить автотест".
> 
> Если это не помогло, попробуйте установить блокировщик рекламы на браузер, например adblock, затем выключить расширение tamper monkey, закрыть браузер. Затем включите браузер, включить расширение tamper monkey и попробуйте снова "решить" тест

я просто нажала на f5 и все заработало, спасибо огромное!!!!!!!!!!!!

**Comment by angelinkaaaa** [comment]
скипает ответы, как исправить?

**Comment by angelinkaaaa** [comment]
есть возможность что заблокируют аккаунт? может есть какое-то определенное количество чтобы не блокали?

**Comment by kiray-gd** [comment]
@angelinkaaaa Да, такая возможность есть. Об этом прямо сказано в пользовательском соглашении на сайте нмо
![image](https://github.com/user-attachments/assets/8404704f-42fd-4c8f-ae57-6dae92d848e0)

**Comment by angelinkaaaa** [comment]
> @angelinkaaaaДа, такая возможность есть. Об этом прямо сказано в пользовательском соглашении на сайте нмо ![изображение](https://private-user-images.githubusercontent.com/10156000/389279339-8404704f-42fd-4c8f-ae57-6dae92d848e0.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzI0NTc3MTUsIm5iZiI6MTczMjQ1NzQxNSwicGF0aCI6Ii8xMDE1NjAwMC8zODkyNzkzMzktODQwNDcwNGYtNDJmZC00YzhmLWFlNTctNmRhZTkyZDg0OGUwLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDExMjQlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQxMTI0VDE0MTAxNVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTQ2YjdhNDYzYzY3ZGYwMzVmNzQxNjhiYzY5ZTM1ODk3ZTYxNmU4ZDYyZGNjYWQwNWVlMDRmMjljNmZhMjQ5YjkmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.hOPSUFZFMA6S8A_p-EBlLJSSBw1lkDxWEFKtTplQM3A)

есть какое-то количество баллов, которое можно набирать, для предоосторожности?

**Comment by kiray-gd** [comment]
@angelinkaaaa наверное такой информации нет ни у кого

**Comment by Levin-crk** [comment]
не выдает вариант ,не знаете как можно решить проблему ?

**Comment by TemPoun** [comment]
не работает

**Comment by TemPoun** [comment]
скачал все по гайду и ничего нету

**Comment by TemPoun** [comment]
мне бы сделать 11 штук а тут капец

**Comment by TemPoun** [comment]
кого-то банили уже за автотесты?

**Comment by Levin-crk** [comment]
> @angelinkaaaa наверное такой информации нет ни у кого

Привет , а помнишь ты делал автотест с вставкой ссылки , есть варик его починить ?

**Comment by TemPoun** [comment]
левик ты делаешь тесты,не банит? если забанят что будет

**Comment by Levin-crk** [comment]
> левик ты делаешь тесты,не банит? если забанят что будет

если забанят , то все , сильно не гони и все

**Comment by TemPoun** [comment]
они же могли написать что за это будет бан,но банить они не будут

**Comment by Levin-crk** [comment]
> левик ты делаешь тесты,не банит? если забанят что будет

у тебя варианты выдает ?

**Comment by TemPoun** [comment]
да все норм

**Comment by TemPoun** [comment]
смысле сильно не гони

**Comment by TemPoun** [comment]
?

**Comment by TemPoun** [comment]
они проверяют тесты по минутам завершения его?

**Comment by HeisPovedim** [comment]
ну че там, скрипт работает у всех? после загрузки вопросов перекидывает на страницу с вопросами, в консоле приходит нормально объект, где вопрос - ответ, но при этом console.log говорит что вопрос не найден
